### PR TITLE
Updating Readme.md

### PR DESCRIPTION
**Pull request by nehajain1809**
Signed-off-by: nehajain1809 <nehajain@Nehas-MacBook-Pro.local>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,91 @@
-# Podman Ansible Module
 
+Ansible Module for Podman
+=========================
+This module will allow to use Podman to manage the images.
```
Comment (rhatdan) [inline README.md]: The Ansible Podman module allows Ansible to use Podman to manage container images.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,91 @@
...
-## License
+Requirements
+============
+* Python 3.6+
+* Ansible 2.8
 
-Ansible Podman module is licensed under Apache 2.0 license. See the [LICENSE](LICENSE) file for details.
+Installation
+============
+1. Install [Podman](https://github.com/containers/libpod)
```
Comment (rhatdan) [inline README.md]: This would probably be a better link:
https://github.com/containers/libpod/blob/master/install.md

**Comment by rhatdan** [comment]
@tomsweeneyredhat PTAL

**Review by TomSweeneyRedHat — COMMENTED** [review]
I concur with the changes @rhatdan suggests, otherwise LGTM

File: README.md (hunk 1 above)
Comment (nehajain1809) [reply README.md]: Updated the README.md with the suggested changes. Thank you.

File: README.md (hunk 2 above)
Comment (nehajain1809) [reply README.md]: Updated the link. Thank you.

**Pull request merged**